notebooks/COS/DataDl/DataDl.ipynb

```diff
@@ -160,7 +160,7 @@
     "\n",
```

Comment:
<p>Great job, Sierra! Thank you for making the changes. I wanted to bring your attention to the Figures 1.2, 1.5 and 1.6 are not displaying properly. Upon further inspection, I noticed that the source references for these images are pointing to the "Revised-Figures" directory, which appears to be missing. Additionally, it's always a good practice to include an <strong>alt </strong>attribute for the &lt;img&gt; tag, which provides valuable information in case image cannot be displayed or is inaccessible. This can greatly improve accessiblity and user experience. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>